### PR TITLE
Replace vManage with Manager

### PR DESCRIPTION
**Pull request by kagrski**
# Pull Request summary:
Replace vManage with Manager

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,22 +4,22 @@
 
 [![Python-Supported](https://img.shields.io/static/v1?label=Python&logo=Python&color=3776AB&message=3.8%20|%203.9%20|%203.10%20|%203.11%20|%203.12)](https://www.python.org/)
 
-vManage client is a package for creating simple and parallel automatic requests via official vManage API. It is intended to serve as a multiple session handler (provider, provider as a tenant, tenant). The library is not dependent on environment which is being run in, you just need a connection to any vManage.
+Cisco Catalyst SD-WAN SDK is a package for creating simple and parallel automatic requests via official Manager API. It is intended to serve as a multiple session handler (provider, provider as a tenant, tenant). The library is not dependent on environment which is being run in, you just need a connection to any Manager.
```
Comment (sbasan) [inline README.md:7]:
```suggestion
Cisco Catalyst WAN SDK is a package for creating simple and parallel automatic requests via official Manager API. It is intended to serve as a multiple session handler (provider, provider as a tenant, tenant). The library is not dependent on environment which is being run in, you just need a connection to any Manager.
```

File: README.md [hunk 2]
```diff
@@ -4,22 +4,22 @@
...
 
 ## Installation
 ```console
 pip install catalystwan
 ```
 
 ## Session usage example
-Our session is an extension to `requests.Session` designed to make it easier to communicate via API calls with vManage. We provide ready to use authenticetion, you have to simply provide the vmanage url, username and password as as if you were doing it through a GUI. 
+Our session is an extension to `requests.Session` designed to make it easier to communicate via API calls with Manager. We provide ready to use authenticetion, you have to simply provide the Manager url, username and password as as if you were doing it through a GUI.
```
Comment (sbasan) [inline README.md:15]:
```suggestion
Our session is an extension to `requests.Session` designed to make it easier to communicate via API calls with SDWAN Manager. We provide ready to use authentication, you have to simply provide the Manager url, username and password as as if you were doing it through a GUI. 
```

File: README.md [hunk 3]
```diff
@@ -4,22 +4,22 @@
...
+Our session is an extension to `requests.Session` designed to make it easier to communicate via API calls with Manager. We provide ready to use authenticetion, you have to simply provide the Manager url, username and password as as if you were doing it through a GUI. 
 ```python
-from catalystwan.session import create_vManageSession
+from catalystwan.session import create_manager_session
 
 url = "example.com"
 username = "admin"
 password = "password123"
-session = create_vManageSession(url=url, username=username, password=password)
+session = create_manager_session(url=url, username=username, password=password)
```
Comment (sbasan) [inline README.md:22]: Use context manager as suggested method of using session
```suggestion
with create_manager_session(url=url, username=username, password=password) as session:
    session.get("/dataservice/device")
```
Explain that `close()` method needs to be executed to release user session resource on SDWAN manager, when not using context manager.
 ```suggestion
session = create_manager_session(url=url, username=username, password=password)
session.get("/dataservice/device")
session.close()
```

**Review by cicharka — APPROVED** [review]

**Review by sbasan — APPROVED** [review]
lgtm

**Pull request merged**